ctapipe/reco/sklearn.py

```diff
@@ -788,7 +788,8 @@ def predict_table(self, key, table: Table) -> Tuple[Table, Table]:
         fov_lon = table["hillas_fov_lon"] + disp * np.cos(psi)
         fov_lat = table["hillas_fov_lat"] + disp * np.sin(psi)
 
-        # For now: Assume parallel pointing for each run
+        # FIXME: Assume parallel pointing for each run
+        self.log.warning("Assuming parallel pointing for each run")
```

Comment:
Assuming "constant and parallel" pointing, right?

Reply:
The pointing is loaded from the observation block. How would non-constant pointing be handled there?